### DISCUSSION TITLE
What about porting BLIP (by SalesForce)?

### DISCUSSION THREAD

**Discussion by Sovenok-Hacker** [ggml-org/ggml#540] [Ideas]
What about porting BLIP (by SalesForce)?

**Comment by cmp-nct** [comment]
There is clip.cpp (https://github.com/monatis/clip.cpp), though it's a Linux only variant that breaks a bit with typical ggml programming (latest c++ standards, no cross compatibility)
I'd guess giving it proper cross compatibility wouldn't be a huge task and BLIP is quite related to CLIP so that's probably a good starting point.
